### PR TITLE
custom 404 page

### LINKED ISSUE #142: [Template] 404 Error Page
The current live site just redirects you to the home page when there is a 404 error.   We should consider adding a custom 404 page.

### PR DESCRIPTION
**Pull request by samuelopra**
Fix #142 

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.live/testing
- After: https://issue-142--hubblehomes-com--aemsites.hlx.live/testing

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -1,67 +1,121 @@
...
+      type="module"
+      crossorigin="use-credentials"
+    ></script>
+    <script src="/scripts/aem.js" type="module"></script>
+    <link rel="stylesheet" href="/styles/styles.css" />
+    <style>
+      main.error {
+        min-height: calc(100vh - var(--nav-height));
+        display: flex;
+        align-items: center;
```
Comment (bhellema) [inline 404.html:23]: Remove the center alignment for the items and it should bring the items to the top of the page as per the discussion with Nate.

```suggestion
```

Comment (samuelopra) [reply 404.html]: changed

**Comment by bhellema** [comment]
Can we scale the svg size smaller for mobile so all icons can fit in the display?

Screenshot_20240709_184043_Chrome

**Comment by samuelopra** [comment]
> Can we scale the svg size smaller for mobile so all icons can fit in the display?
> 
> Screenshot_20240709_184043_Chrome

should be better now

**Comment by codevoyage** [comment]
From what I recall, Nate preferred the version where the page content is closer to the top rather than centred vertically.

**Comment by samuelopra** [comment]
> From what I recall, Nate preferred the version where the page content is closer to the top rather than centred vertically.

@codevoyage @bhellema this is fixed , for some reason the old change was cached, so switching the path updated the dom

**Review by varunmitra — COMMENTED** [review]
For the mobile view icons are aligned horizontally rather than vertically. I think, what @bhellema was referring to was making the icon size smaller while keeping icons in the vertical layout. On a mobile view icons appear too close to each other

image

**Review by varunmitra — APPROVED** [review]

**Pull request merged**